libs/wingc/src/jsify/snapshots/capture_identifier_from_preflight_scope.snap

```diff
@@ -43,26 +43,28 @@ class $Root extends $stdlib.std.Resource {
     class $Closure1 extends $stdlib.std.Resource {
       constructor(scope, id, ) {
         super(scope, id);
-        this._addInflightOps("handle", "$inflight_init");
```

Comment:
Is this a regression?

Reply:
Fortunately no - there's a new _getInflightOps() method generated at the end of the snapshot instead